st4/__init__.py

```diff
@@ -39,6 +39,11 @@ def can_start(
         workspace_folders: List[WorkspaceFolder],
         configuration: ClientConfig
     ) -> Optional[str]:
+        rootPath = configuration.init_options.get('rootPath')
+        rootUri = configuration.init_options.get('rootUri')
```

Comment:
I'm unsure why `rootPath` and `rootUri` would be part of the initializationOptions. I think this can just be removed?